### DISCUSSION TITLE
Copilot

### DISCUSSION THREAD

**Discussion by Drogaria01** [community/community#66739] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I eant  have several app ideas that I will put to work. How and where can I find trustworthy professionals in writing the code for applications?

**Comment by arafajs** [comment]
- **Online job boards:** There are many online job boards where you can post your project and find developers who are interested in working on it. Some popular options include Upwork, Fiverr, and Toptal.
- **Networking:** Talk to your friends, family, and colleagues to see if they know any developers who might be a good fit for your project. You can also attend industry events or meetups to meet new people.
- **Contacting agencies:** There are many agencies that specialize in matching businesses with developers. These agencies can help you find the right developer for your project and manage the development process.
- **Hiring a freelancer:** If you only need a small amount of work done, you can hire a freelancer to do the coding for you. There are many freelancers available online, and you can find them through online job boards or freelancing platforms.

**Comment by ghostinhershell** [comment]
Hi @Drogaria01, we're happy you're here 😀

We currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities.

With that being said, please note that he Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Cheers 👋🏾
